python_tools.spec

```diff
@@ -25,8 +25,8 @@ Requires: py2-bottleneck
 Requires: py2-downhill 
 Requires: py2-theanets
 Requires: py2-xgboost
-Requires: py2-llvmlite
-Requires: py2-numba
+#Requires: py2-llvmlite
```

Comment:
@smuzaffar this seems to trigger a couple of failures in the unit tests of PythonAnalysis since CMSSW_10_4_X_2018-10-11-2300 because these tools are not found in the import. Is this needed?